Yoast/wordpress-seo
# Make the plugin sidebar title depend on the plugin version.

## Summary

This PR can be summarized in the following changelog entry:

* Make the plugin sidebar title correctly reflect the Free or Premium version


## Test instructions
- this PR should be merged on Premium and tested there or, alternatively:
- change the line
`'isPremium'                 => WPSEO_Utils::is_yoast_seo_premium(),`
to `'isPremium' => true,` or `'isPremium' => false,`
- check the correct plugin name "Yoast SEO" or "Yoast SEO Premium" is displayed in:
  - the sidebar top bar
  - the pinned icon tooltip
  - the plugin name in the Gutenberg "more menu" under the "Plugins" section 

As far as I can tell, these titles are used only for displaying purposes and shouldn't affect in any way the plugin / sidebar registration.

## UI changes

Fixes https://github.com/Yoast/wordpress-seo-premium/issues/2228
